### DISCUSSION TITLE
Table with group adds a row at the page break

### DISCUSSION THREAD

**Discussion by drydenem** [eclipse-birt/birt#1463] [Q&A]
I have a table with a group. I have two columns in the header of group that drop all. At the page break, there is an extra row, but the only the header columns styles/data go in those rows? Any ideas on how I can get rid of this extra row? See in this image, in the last row, the numbers stop, but the two columns that are in the group header fill their style. [](url)
<img width="405" alt="Screenshot 2023-10-27 at 4 49 24 PM" src="https://github.com/eclipse-birt/birt/assets/102245348/02fa8dc1-0ef3-4bb9-bddd-81f4818ff1b4">
<img width="405" alt="Screenshot 2023-10-27 at 4 49 24 PM" src="https://github.com/eclipse-birt/birt/assets/102245348/26db3bd5-945e-4e27-9598-681caafdad2e">

**Comment by hvbtup** [comment]
I don't see the use for Drop: All. Maybe the issue can be avoided with Drop: Detail?
And did you delete superfluous group footer lines?
